tools/getRollupData/getRollupData.ts

```diff
@@ -31,7 +30,12 @@ async function main() {
     // Filter first initialization (deployment)
     const filter = rollupManager.filters.AddExistingRollup(1);
     const eventsAddRollup = await rollupManager.queryFilter(filter, 0, "latest");
-    const upgradeToULxLyBlockNumber = eventsAddRollup[0].blockNumber;
+    let upgradeToULxLyBlockNumber;
+    if (eventsAddRollup.length > 0) {
+        upgradeToULxLyBlockNumber = eventsAddRollup[0].blockNumber;
+    } else {
+        console.log("No event AddExistingRollup");
```

Comment:
Then, `upgradeToULxLyBlockNumber` will remain `undefined`
I will setup the same as the `deploymentRollupManagerBlockNumber`